config/kernel-signal-stop.m4

```diff
@@ -0,0 +1,21 @@
+dnl #
+dnl # 4.4 API change
+dnl # Added kernel_signal_stop
+dnl #
+AC_DEFUN([ZFS_AC_KERNEL_SRC_SIGNAL_STOP], [
```

Comment:
You'll need to call these new macros from `ZFS_AC_KERNEL_TEST_SRC` and `ZFS_AC_KERNEL_TEST_RESULT` in `kernel.m4`.